python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -163,42 +176,31 @@ def to_default_asset_events(
             and is_node_successful
             and not is_node_ephemeral
         ):
-            started_at = dateutil.parser.isoparse(event_node_info["node_started_at"])
-            finished_at = dateutil.parser.isoparse(event_node_info["node_finished_at"])
-            duration_seconds = (finished_at - started_at).total_seconds()
+            metadata = {
+                "materialization": node_materialization,
```

Comment:
this is a dbt concept?  do you think we might need to rename the key to disambiguate?

Reply:
This is a SQL concept (e.g. you can materialize as a `table`, `view`, `materialized view`, etc). dbt also augments this concept by adding `incremental` as a "materialization".

Re: disambiguation keys -- we should just come up with some scheme in a later PR that adds a prefix to these metadata fields that are automatically populated by our integration.

Reply:
Agreed that `table`, `view`, and `materialized view` are SQL concepts, but referring to the enum of these as `materialized` is a dbt-specific way of looking at things. IMO using that term is confusing without qualifying it with "dbt" in some way, especially because "materialize" has its own meaning in Dagster which is at least subtly different.

Reply:
Fair! I'll rename it to `dbt_materialization` for now. We could refactor this later down the line once we get some alignment on structuring the metadata fields for our integrations.